gammapy/datasets/make.py

```diff
@@ -64,3 +71,121 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
```

Comment:
You have to somehow tell the user what the choices are.
This is complicated in this case because `observatory_locations` doesn't show up in the API docs (see issue #111), so you can't refer to there from here.
So maybe all we can do for now is leave a TODO in the docstring and mention `observatory_locations` without linking to it?